AY2021S2-CS2103T-T11-2/tp
# Simplify storage by merging addressbook and projectsfolder

closes #162, closes #140

This will allow us to
1. Reduce unnecessary duplication of code.
2. Reduce complexity.
3. Not have to change architecture diagram in DG.

Tasks:
- [x] Merged addressbook and projectsfolder
- [x] Fix automated tests
- [x] Check for leftover addressbook or projectsfolder in code
- [x] Manual tests to ensure functionality

Other changes
1. TypicalProjects#getTypicalProjects no longer throws an exception
2. Logger changed from addressbook.log to colab.log

Manual tests:
1. Run through one command of each type, check safe file, exit, launch again
2. Invalid file test.
3. Safe file not found test.
